ConsoleApp/Models/Context.cs

```diff
@@ -0,0 +1,39 @@
+namespace ConsoleApp.Models
+{
+    public class Context
+    {
+        private List<Record> Records = new List<Record>();
+        public decimal Balance { get { return Records.Sum(x => x.Summ); } }
+        public List<string> Operations { get { return Records.Select(x => x.ToString()).ToList(); } }
+
+        public void Deposit(decimal summ)
+        {
+            Records.Add(new Record() { Summ = summ, Id = Records.Count });
+        }
+
+        public void Withdraw(decimal summ)
+        {
+            Records.Add(new Record() { Summ = -summ, Id = Records.Count });
+        }
+        public Context()
+        {
+        }
+        // summ - сума початкового депозита
+        public Context(decimal summ)
```

Comment:
not critical, but this constructor is not needed, it's better to make deposit explicitly after initialization in this case